src/meta/processors/admin/AdminClient.cpp

```diff
@@ -346,7 +368,7 @@ folly::Future<Status> AdminClient::getResponse(const HostAddr& host,
        respGen = std::move(respGen),
        this]() mutable {
         auto client = clientsMan_->client(host, evb);
-        remoteFunc(client, std::move(req))
+        remoteFunc(host, client, std::move(req))
```

Comment:
If this changes is only for logging, why not just print them in related interfaces?

Reply:
👌